src/LinearMath/btIDebugDraw.h

```diff
@@ -166,9 +166,9 @@ class	btIDebugDraw
 	virtual void drawTransform(const btTransform& transform, btScalar orthoLen)
 	{
 		btVector3 start = transform.getOrigin();
-		drawLine(start, start+transform.getBasis() * btVector3(orthoLen, 0, 0), btVector3(1.f,0.3,0.3));
-		drawLine(start, start+transform.getBasis() * btVector3(0, orthoLen, 0), btVector3(0.3,1.f, 0.3));
-		drawLine(start, start+transform.getBasis() * btVector3(0, 0, orthoLen), btVector3(0.3, 0.3,1.f));
```

Comment:
in general case

float(1.0) != double(1.0)

double(float(1.0)) equal something like 1.000000000000023 or 0.999999999999981

see for example this discussion https://stackoverflow.com/questions/2386772/difference-between-float-and-double

i do not know what best way for this change....